apps/www/components/ActionBar/index.js

```diff
@@ -382,20 +404,9 @@ const ActionBar = ({
       href: shareUrl,
       onClick: (e) => {
         e.preventDefault()
-        trackEvent(['ActionBar', 'share', shareUrl])
-        if (inNativeApp) {
-          postMessage({
-            type: 'share',
-            payload: {
-              title: document.title,
-              url: shareUrl,
-              subject: emailSubject,
-              dialogTitle: t('article/share/title'),
-            },
-          })
+        handleShareClick()
+        if (isNativeApp) {
           e.target.blur()
```

Comment:
it kinda bothers me that the if clause in the `handleShareClick` function is the same as the one to blur the event target (`if (inNativeApp)`).
I would pass the event/event.target to `handleShareClick`

Reply:
yeah i passed it in before, don't know why i removed it on a later refactor...

Reply:
done